src/pages/workspace/accounting/qbd/QuickBooksDesktopSetupPage.tsx

```diff
@@ -78,7 +78,7 @@ function RequireQuickBooksDesktopModal({route}: RequireQuickBooksDesktopModalPro
         },
     });
 
-    const shouldShowError = hasError;
+    const shouldShowError = true;
```

Comment:
hmmm this doesn't seem right. Surely this variable is either redundant or incorrect